projects/views.py

```diff
@@ -1,3 +1,7 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from .models import Project
 
-# Create your views here.
+
+def list_projects(request):
+    projects = Project.objects.all()
```

Comment:
I think this is the ideal usecase for https://docs.djangoproject.com/en/1.8/ref/class-based-views/generic-display/#listview